### ISSUE TITLE
list of indices for face is not supported?

### ISSUE THREAD

**Issue by ComicAddict** [playcanvas/model-viewer#302]
When a PLY file with list of vertex indices as for defining a face is fed for the online demo I get the following error:

![image](https://github.com/user-attachments/assets/f731d53c-1832-414e-b3a9-0b874dda7744)

The header of the file is the following:

```
ply
format binary_little_endian 1.0
element vertex 10
property double x
property double y
property double z
property uchar r
property uchar g
property uchar b
element edge 2
property int vertex1
property int vertex2
property double w1
property double w2
element face 10
property list int int vertex_index
property uchar red
property uchar green
property uchar blue
property double v
end_header
```

Same file can is readable with both Blender and MeshLab.

**Comment by slimbuck** [comment]
Hi @ComicAddict ,

Model-viewer currently supports PLY files containing 3d gaussian splatting scenes only.

For models we generally rely on glb/gltf.

May I ask why you're using PLY for meshes? We've not had a request to support PLY meshes before.

Thanks!

**Comment by ComicAddict** [comment]
Hi @slimbuck ,

I was checking out all the possible PLY viewers to see if there are some tools that can view the custom attributes. I have PLY files with custom per vertex, per face and per edge information. Are there any other file format that can support per face and per edge custom data?

**Comment by slimbuck** [comment]
Yes PLY is perfect for custom data and the only format I know of that supports this.

How would you expect the viewer to display the custom data? Show the values in a table or actually render the data on the 3d viewport?
